tests/components/hydrawise/test_init.py

```diff
@@ -26,32 +21,3 @@ async def test_setup_import_success(hass: HomeAssistant, mock_pydrawise: Mock) -
         HOMEASSISTANT_DOMAIN, "deprecated_yaml_hydrawise"
     )
     assert issue.translation_key == "deprecated_yaml"
-
-
-async def test_connect_retry(
```

Comment:
Why did you remove these tests?

`coordinator.async_config_entry_first_refresh()` can raise an `ConfigEntryNotReady`